pkg/cloudprovider/aws/instancetype.go

```diff
@@ -135,9 +138,18 @@ func (i *InstanceType) memory() resource.Quantity {
 	)
 }
 
-// Setting ephemeral-storage to be arbitrarily large so it will be ignored during binpacking
+// Setting ephemeral-storage to be either the default value or what is defined in blockDeviceMappings
 func (i *InstanceType) ephemeralStorage() resource.Quantity {
-	return resource.MustParse("100Pi")
+	EphemeralBlockDevice := amifamily.GetAMIFamily(i.provider.AMIFamily, &amifamily.Options{}).EphemeralBlockDevice()
```

Comment:
Local variables should use lower camel case. UpperCamel is reserved for globals and public interfaces
```suggestion
	ephemeralBlockDevice := amifamily.GetAMIFamily(i.provider.AMIFamily, &amifamily.Options{}).EphemeralBlockDevice()
```

Reply:
oops, that was an oversight.  Fixing.